lib/ex_sdp/origin.ex

```diff
@@ -75,6 +85,29 @@ defmodule ExSDP.Origin do
     end
   end
 
+  # a workaround to support SDP sent by Tapo C320WS camera which puts an additional junk byte in the origin attribute
+  defp do_parse(origin, 7) do
+    with {:ok, [username, sess_id, sess_version, _junk, nettype, addrtype, address]} <-
+           Utils.split(origin, " ", 7),
+         {:ok, addrtype} <- Address.parse_addrtype(addrtype),
+         {:ok, address} <- Address.parse_address(address) do
+      # check whether fqdn
+      address = if is_binary(address), do: {addrtype, address}, else: address
+
+      origin = %__MODULE__{
+        username: username,
+        session_id: String.to_integer(sess_id),
+        session_version: String.to_integer(sess_version),
+        network_type: nettype,
+        address: address
+      }
+
+      {:ok, origin}
+    else
+      {:error, _reason} = error -> error
+    end
+  end
```

Comment:
`do_parse` could accept already split `origin`, so you can reuse the rest of the parsing